docs/managed-updates.md

```diff
@@ -7,109 +7,117 @@ RFC 2119.
 
```

Comment:
since we're lower casing all MUST/MUST NOT etc key words, can you remove the section above this line as well?